src/Service/ExcelImport.php

```diff
@@ -271,4 +312,27 @@ private function getCellValueOrNull(Worksheet $sheet, int $col, int $row)
 
         return $cell->getValue();
     }
+
+    private function checkRemovedElements($doc, $array, $type)
```

Comment:
A better name than `$array` would be good here (is this something like $keepIdentifierList ?).

Reply:
this is an array of items or associations from the spreadsheet file, every item/association not present in this array will be deleted from the database. Due `$existingItems` refer to elements from the database do you think `$keepIdentifierList` would be a good name? or something like `$keepElementsList`?

Reply:
Either name sounds fine as it lets one know a bit about what it is for.  `$array` tells us almost nothing.